src/repair.c

```diff
@@ -86,9 +92,27 @@ static bool parse_opt(int opt, char *optarg)
 		return true;
 	case 'q':
 		cmdline_option_quick = true;
+		/* mutually exclusive flags */
+		if (cmdline_option_picky) {
+			error("You cannot use --quick and --picky together\n\n");
```

Comment:
why we can't use --quick and --picky together?

Reply:
because the purpose of --quick is to only add missing files...  And the process of a verify is this:

1) Add missing files
(with --quick) the process is finished here
2) Fix corrupt files
3) Remove files that were deleted in manifests but are still in the system
4) remove files that are not in any manifest

So with --quick and --picky nothing would happen cause the code would never reach the picky part anyway, so it is confusing.

Reply:
it is actually how it works today, but it is wrong:
```
$ sudo swupd diagnose --quick --picky
Diagnosing version 30440

Inspected 673920 files

Diagnose successful
```

Reply:
In my mind quick was suppose to do both. Reading the doc looks like you are correct. Merging this now then.